db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
+    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
```

Comment:
Yeah you dont want repository.save to cause save happening in pulp. This is the safe way :). Infact there is a precedent for this look at https://github.com/Katello/katello/blob/master/db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb#L4